src/github.html

```diff
@@ -0,0 +1,55 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Seattle Tech Bloc - Open Source Projects</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Seattle Tech Bloc directory of open source projects">
+    <meta name="author" content="Tech Bloc SEA">
+
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+    <main class="container">
+        <h1>Seattle Tech Bloc</h1>
+
+        <img width="100em" src="/github.svg" alt="GitHub" />
+
+        <p>
+            Contribute to our open source projects
+        </p>
+
+        <a href="https://github.com/SeattleDSA/1-312-hows-my-driving/">
+            <p>1-312-hows-my-driving</p>
+            <p class="subtitle">A Flask frontend to the SPD Lookup service</p>
+        </a>
+        <a href="https://github.com/SeattleDSA/spd-lookup/">
+            <p>SPD Lookup Service</p>
+            <p class="subtitle">The Go backend to 1-312-hows-my-driving</p>
+        </a>
+        <a href="https://github.com/SeattleDSA/OpenOversight/">
+            <p>OpenOversight</p>
+            <p class="subtitle">A fork of the Lucy Parsons Labs project for the Salish Sea</p>
+        </a>
+        <a href="https://github.com/SeattleDSA/pra-request-tracker/">
+            <p>Public Records Act Request Tracker</p>
+            <p class="subtitle">An original project for tracking PRA requests</p>
```

Comment:
_Very original, totally unique ideas, definitely not drawing inspiration from anything that already exists_ 😉

Reply:
Hahaha I wrote that as a nice little in joke for us lol!